### ISSUE TITLE
Invalid summary link after adding new task

### ISSUE THREAD

**Issue by lebe-dev** [alisaifee/jira-cli#24]
Hello

We don't use subdomains for our jira instance (our url: `https://somehost/jira`)

**problem:**
when i add task
`jira-cli new --protocol=rest --type=bug --priority=Minor --project XYZ 'Some task'`

output:
```
status               : Open
reporter             : myuser
summary              : Some task
assignee             :
link                 : https://somehost/browse/XYZ-3
issue                : XYZ-3
```

Result: final link is invalid, because it forgots `/jira` subdir

**Comment by alisaifee** [comment]
Thanks @lebe-dev. Looks like most users of `jira-cli` (including myself) don't click on the link often because this seems to have been broken since the beginning of `v2`! I'll release a fix soon.

**Comment by alisaifee** [comment]
Released in `2.0.14`
